src/common/models/config.model.ts

```diff
@@ -167,3 +168,39 @@ export interface IConfigCustomAction extends IConfigMethod {
   icon: string
   fields: IConfigInputField[]
 }
+
+export interface IConfigPagination {
+  source: 'query' // TODO: add 'headers'
+  type: 'lazy-loading' // TODO: add 'buttons'
```

Comment:
I would add `buttons` from day 1, and use them as default.

Reply:
Yes, I'll do that.